src/components/dev-hub/learn-tabs.js

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { Tabs as LeafyTabs, Tab } from '@leafygreen-ui/tabs';
+import Button from './button';
+
+const TabButtonStyle = () => css`
+    border-radius: 0;
+    :active,
+    :hover,
+    :focus {
+        color: ${({ theme }) => theme.colorMap.darkGreen};
+        :after {
+            background-color: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const StyledTabButton = styled(Button)`
+    ${TabButtonStyle}
+`;
```

Comment:
```suggestion
const StyledTabButton = styled(Button)`
    border-radius: 0;
    :active,
    :hover,
    :focus {
        color: ${({ theme }) => theme.colorMap.darkGreen};
        :after {
            background-color: ${({ theme }) => theme.colorMap.darkGreen};
        }
    }
`;
```

Similar thought here, could we put the styles directly into `styled`? Then `theme` will work as we expect here since it is always passed to `styled` and not `css`

Reply:
I like this approach. Will make changes.